samples/bmp280-sensor-azure-iot-hub/Readme.md

```diff
@@ -0,0 +1,180 @@
+
+## Prepare the hardware
+
+Use the hardware components to build the circuit as depicted in the following diagram:
+
+![Raspberry Pi 2 with .NET Iot](images/RP-BMP280_bb.png)
+
+## Pins
+
+The following are the connections from the Raspberry Pi to the BME280 breakout:
+
+* 3.3V to VIN OR 3V3 (shown in red)
+* Ground to GND (grey)
+* SDA (GPIO 2) to SDI (blue)
+* SCL (GPIO 3) to SCK (yellow)
+* GPIO 18 to LED anode (longer, positive lead)
+* LED cathode (shorter, negative lead) to 330 Ω resistor (either end)
+* 330 Ω resistor (other end) to ground
+
+## Run the Solution
+
+Use your Azure Subscription for following steps:
+
+1. Make sure your [Pins](#-Pins) are correctly connected.
+2. Create an IoT Hub using your Azure Subscription and leave the authentication options to default (Symmetric Key, autogenerated). You'll need IoT Hub name in the next steps in your code. You can use free tier of IoT Hub as well.
+
+3. Create a `pfx` certificate using OpenSLL following [Create test certificate using OpenSSL](create-certificate.md) documentation.
+
+4. Add your certificate into root folder of the solution like `raspberry-pi.pfx` for demo purposes verified certificate in this sample is removed from repo.
+
+5. Provide your device id and IoT Hub device connection detail into `Program.cs`. Open your solution with Microsoft Visual Studio 2019.
+
+    ```csharp
+    const string DeviceID = "<replace-with-your-device-id>";
+    const string IotBrokerAddress = "<replace-with-your-iot-hub-name>.azure-devices.net";
+    ```
+
+### Building the sample
+
+1. Starting in the folder where you unzipped the samples, go to the subfolder for this specific sample. Double-click the Visual Studio Solution (.sln) file.
+2. Press Ctrl+Shift+B, or select **Build** \> **Build Solution**.
+3. Make sure to click on device explorer and select your device.
+
+The next steps depend on whether you just want to deploy the sample or you want to both deploy and run it.
+
+### Deploying the sample
+
+* Select Build > Deploy Solution.
+
+### Deploying and running the sample
+
+* To debug the sample and then run it, press F5 or select Debug >  Start Debugging.
+
+## Remote Debugging
+
+To enable remote debugging on Raspberry Pi make sure following below steps.
+
+* [Remote debugging with VS Code on Windows to a Raspberry Pi using .NET Core on ARM](https://www.hanselman.com/blog/remote-debugging-with-vs-code-on-windows-to-a-raspberry-pi-using-net-core-on-arm)
+
+* [How to use Windows 10's built-in OpenSSH to automatically SSH into a remote Linux machine](https://www.hanselman.com/blog/how-to-use-windows-10s-builtin-openssh-to-automatically-ssh-into-a-remote-linux-machine)
+
+After the setup we have [launch.json](../.vscode/launch.json) file to enable build and publish steps. Also we have [tasks.json](../.vscode/tasks.json) . file to enable remote debugging feature helps us to debug our application on Raspberry Pi on Raspberry Pi OS with .NET Core IoT.
```

Comment:
Would be great to add as well for Visual Studio. A lot of developers are directly using Visual Studio.

Reply:
This is added as well.